In [1]:
import pandas as pd

In [12]:
cd C:\Users\Administrator\Desktop\internship

C:\Users\Administrator\Desktop\internship


In [14]:
email = pd.read_csv("sample_email.csv",encoding='latin1')

In [16]:
email.head()

first_name last_name            company_name               address  \
0      James      Butt       Benton, John B Jr    6649 N Blue Gum St   
1  Josephine   Darakjy   Chanay, Jeffrey A Esq   4 B Blue Ridge Blvd   
2        Art    Venere     Chemel, James L Cpa  8 W Cerritos Ave #54   
3      Lenna  Paprocki  Feltz Printing Service           639 Main St   
4    Donette    Foller     Printing Dimensions          34 Center St   

          city      county state    zip        phone1         phone  \
0  New Orleans     Orleans    LA  70116  504-621-8927  504-845-1427   
1     Brighton  Livingston    MI  48116  810-292-9388  810-374-9840   
2   Bridgeport  Gloucester    NJ   8014  856-636-8749  856-264-4130   
3    Anchorage   Anchorage    AK  99501  907-385-4412  907-921-2010   
4     Hamilton      Butler    OH  45011  513-570-1893  513-549-4561   

                           email  \
0              jbutt@luminate.ai   
1  josephine_darakjy@luminate.ai   
2                art@luminate.ai   
3          lpaprocki@luminate.ai   
4     donette.foller@luminate.ai   

                                             subject  \
0  When EMVX creation comes the message "The crea...   
1   It falters when processing highly segmented p...   
2         Cwvemx files falter in Sales firstclass 20   
3   When creating EM / EMVX with Windows 7, it ma...   
4   Drivers for the USB Process Manager could not...   

                                          Email Boby  
0  I have a nearly unreadable font in my Warehous...  
1  On Saturday the version WAREHOUSE Sales firstc...  
2  In export mode, the font in the small preview ...  
3  On my specificationsheet, the text blocks are ...  
4  Since your update come strange bubbles on my s...

In [18]:
email.isnull().sum()

first_name      0
last_name       0
company_name    0
address         0
city            0
county          0
state           0
zip             0
phone1          0
phone           0
email           0
subject         0
Email Boby      0
dtype: int64

## 1.Fetch the Detail like name, email, email Body, email subject and create email template for each individuals. 

- Email Template Sample
  - first_name: James,last_name: Butt, company_name: Benton, John B Jr,	address:6649 N Blue Gum St,city: New Orleans
  - county: Orleans,state: LA,zip: 70116,phone1: 504-621-8927,phone: 504-845-1427,email: jbutt@luminate.ai,
  - subject: When EMVX creation comes the message "The creation was canceled for hardware reasons"
  - ,Email Boby: I have a nearly unreadable font in my Warehouse Software under Export-File Options for 2 days.
    Please do contact me at 504-621-8927.  

## 2. Check and fetch email address from the email body section if any present and store it in another CSV. 
## 3. Check and fetch mail number from the email body section if any present and store it in another CSV.
